docs/views/spec/spec.mdx

```diff
@@ -39,11 +44,13 @@ The Specification is defined as follows:
 `}
 </CodeBlock>
 
+Each of `data`, `label`, and `output` can be either a [ViewElement](/docs/views/spec/other) or a [TableElement](/docs/views/spec/other). If you want to display the data, label, and output as a table, you must use [TableElement](/docs/views/spec/other) entries for each of them.
```

Comment:
This might be a bit confusing. Table Elements should only be used with displayType table since they add `<tr>` tags.

Maybe:

Each of `data`, `label`, and `output` can be a [ViewElement](/docs/views/spec/other). If you want to display the data, label, and output as a table, you must use `displayType="table"` and use [TableElement](/docs/views/spec/other) entries for each of `data`, `label`, and `output` instead.